frontend/src/App.js

```diff
@@ -23,13 +53,52 @@ function App() {
 						<Route path="/login">
 							<LoginPage />
 						</Route>
-            <Route path="/">
-							<HomePage />
+						<Route path="/teacherTasks">
+						   <TeacherTasksPage/>
+						</Route>
+						<Route path="/teacherTaskViewer">
+							<TeacherTaskViewerPage/>
+						</Route>
+						<Route path="/teacherTaskEditor">
+							<TeacherTaskEditorPage/>
+						</Route>
+            			<Route path="/">
+							<HomePage/>
 						</Route>
 					</Switch>
 				</div>
 			</BrowserRouter>
 		</div>
+// 	<BrowserRouter>
+// 		<Switch>
+// 			<div className="App container">
+// 				<AppBar position="static">
+// 					<Toolbar>
+// 						<Typography variant="h5" className={classes.title} component={Link} to="/">
+// 							Quantum Circuit Simulator Group 2
+// 						</Typography>
+// 							<Button className={classes.signup} color="inherit" component={Link} to="/signup"> Sign Up</Button>
+// 							<Button color="inherit" component={Link} to="/login">Login</Button>
+// 					</Toolbar>
+// 				</AppBar>
+// 				<Switch>
+// 					<Route path="/signup">
+// 						<SignupPage />
+// 						<Footer />
+// 					</Route>
+// 					<Route path="/login">
+// 						<LoginPage />
+// 						<Footer />
+// 					</Route>
+// 					<Route path="/">
+// 						{/* <HomePage /> */}
+// 						<QuantumSimulator />
+// 						<Footer />
+// 					</Route>
+// 				</Switch>
+// 			</div>
+// 		</Switch>
+// 	</BrowserRouter>
```

Comment:
Same note as the other comment, why are we trying to merge blocks of dead code?

Reply:
I was using it as a guide when doing the routing. We can delete now.

Reply:
Deleted this when resolving conflict.